src/gui/components/tabs/tabs.styles.js

```diff
@@ -0,0 +1,8 @@
+import { css } from 'glamor'
+
+const layout = css({
+  position: 'relative',
+  border: '1px solid #ddd',
```

Comment:
this not only about user font size.
this is also about making the component reusable.

if you want your tabs to be big, you just have to use them and change font-size with the `em` hack : 
  <Tabs style={{ fontSize: '2em' }} />

==> I tabs that are 2 times bigger, border included (em) or not (px).